west.yml

```diff
@@ -23,7 +23,7 @@ manifest:
       remote: memfault
       revision: 1.17.0
     - name: orb-messages
-      revision: d4b4f8c86dfe88cb48ffca186dab66b9530cc9e7
+      revision: 3dffed6e01fa4aaca347eca52be87bfc298508eb
```

Comment:
can you also update `priv-orb-messages` below?
we just merged an open PR with @TheButlah